buildenv/jenkins/omrbuild.groovy

```diff
@@ -0,0 +1,357 @@
+                'buildDir' : cmakeBuildDir,
+                'configureArgs' : '-Wdev -DCMAKE_C_COMPILER=xlc_r -DCMAKE_CXX_COMPILER=xlC_r -DCMAKE_XL_CreateExportList="/usr/vac/bin/CreateExportList -X64" -DOMR_DDR=OFF -C../cmake/caches/Travis.cmake',
+                'compile' : 'export CCACHE_EXTRAFILES="$PWD/omrcfg.h" && make -j8'
+            ]
+        ],
+        'test' : true,
+        'testArgs' : ''
+    ],
+    'linux_390-64' : [
+        'label': 'Linux && 390',
+        'environment' : [
+            'PATH+CCACHE=/usr/lib/ccache/',
+            'GTEST_COLOR=0'
+        ],
+        'ccache' : true,
+        'buildSystem' : 'cmake',
+        'builds' : [
+            [
+                'buildDir' : cmakeBuildDir,
+                'configureArgs' : '-Wdev -C../cmake/caches/Travis.cmake',
+                'compile' : defaultCompile
+            ]
+        ],
+        'test' : true,
+        'testArgs' : ''
+    ],
+    'linux_aarch64' : [
+        'label' : 'Linux && x86 && compile:aarch64',
+        'environment' : [
+            'PATH+CCACHE=/usr/lib/ccache/:/home/jenkins/aarch64/toolchain/bin',
+            'PLATFORM=aarch64-linux-gcc',
+            'CHOST=aarch64-linux-gnu'
+        ],
+        'ccache' : true,
+        'buildSystem' : 'autoconf',
+        'builds' : [
+            [
+                'buildDir' : autoconfBuildDir,
+                'configureArgs' : 'SPEC=linux_aarch64',
+                'compile' : defaultCompile
+            ]
+        ],
+        'test' : false
+    ],
+    'linux_arm' : [
+        'label' : 'Linux && x86 && compile:arm',
+        'environment' : [
+            'PATH+CCACHE=/usr/lib/ccache/:/home/jenkins/arm/toolchain/bin',
+            'PLATFORM=arm-linux-gcc',
+            'CHOST=arm-linux-gnueabihf'
+        ],
+        'ccache' : true,
+        'buildSystem' : 'autoconf',
+        'builds' : [
+            [
+                'buildDir' : autoconfBuildDir,
+                'configureArgs' : 'SPEC=linux_arm',
+                'compile' : defaultCompile
+            ]
+        ],
+        'test' : false
+    ],
+    'linux_ppc-64_le_gcc' : [
+        'label' : 'Linux && PPCLE',
+        'environment' : [
+            'PATH+CCACHE=/usr/lib/ccache/',
+            'GTEST_COLOR=0'
+        ],
+        'ccache' : true,
+        'buildSystem' : 'cmake',
+        'builds' : [
+            [
+                'buildDir' : cmakeBuildDir,
+                'configureArgs' : '-Wdev -C../cmake/caches/Travis.cmake',
+                'compile' : defaultCompile
+            ]
+        ],
+        'test' : true,
+        'testArgs' : ''
+    ],
+    'linux_riscv64' : [
+        'label' : 'Linux && riscv64',
+        'environment' : [
+            'PATH+CCACHE=/usr/lib/ccache/'
+        ],
+        'ccache' : true,
+        'buildSystem' : 'cmake',
+        'builds' : [
+            [
+                'buildDir' : autoconfBuildDir,
+                'configureArgs' : 'cmake -Wdev -C../cmake/caches/Travis.cmake',
```

Comment:
This looks incorrect. I don't think we should have `cmake` in the `configureArgs` judging by how it is setup for other platforms.

Reply:
Good catch. Fixed. @youngar mentioned getting someone to test this out on riscv?